lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -393,7 +393,7 @@ def dump_config
     end
 
     def create_run_script(file_name, file_content)
-      File.open(file_name, 'w') { |f| f.write(file_content) }
+      File.write(file_name, file_content)
```

Comment:
Is open somewhere else, or they change the write function in Ruby?

Reply:
`File` class has a class method version of `write`, which opens, writes and closes file.

Reply:
Moved to https://github.com/HCK-CI/AutoHCK/pull/158